addons/intercom_phone/RscTitles.hpp

```diff
@@ -0,0 +1,28 @@
+class RscPictureKeepAspect;
+class RscTitles
+{
+    class GVAR(ConnectionIndicatorRsc) {
+        idd = -1;
+        movingEnable = 1;
+        duration = 9999999;
+        fadein = 0;
+        fadeout = 0;
+        onLoad = QUOTE(uiNamespace setVariable [ARR_2(QUOTE(QGVAR(ConnectionIndicatorRscDisplay)),_this select 0)];);
+        class controls {
+            class ConnectionIndicator: RscPictureKeepAspect {
+                idc= 1112;
+                type = 0;
+                style = "0x30 + 0x800";
+                colorText[] = { 1, 1, 1, 1 };
+                colorBackground[]={0, 0, 0, 0};
+                font = "PuristaMedium";
+                sizeEx = "(((((safezoneW / safezoneH) min 1.2) / 1.2) / 25) * 0.8)";
+                text=QPATHTOF(ui\tfar_intercom_phone.paa);
+                x="(profilenamespace getvariable [""IGUI_grid_TFAR_Volume_X"",  0.85 * safezoneW + safezoneX])";
```

Comment:
Nice idea to reuse this.
But people can enable volume indicator to always show, that means these will then overlap eachother?
Maybe want to hide the speech volume indicator while you're in intercom

Reply:
Actually you added a grid for intercom phone in config, but here you seem to use the volume one

Reply:
Honestly, I don't understand dialogues and, more specifically, how to position them at all... So was hoping someone would come round and comment on it like you did. So this grid, do I use the class name specified in https://github.com/michail-nikolaev/task-force-arma-3-radio/pull/1530/files#diff-5238f9c6d8630cb9ec6d642487487911310e6619735d338c5ea33bf702fa4aa2R46 ?